cernopendata/modules/fixtures/data/records/delphi-documentation-manuals.json

```diff
@@ -0,0 +1,205 @@
+[
+  {
+    "abstract": {
+      "description": "Detailed description of the DELPHI conditions database contents, and the tools coming with it."
+    },
+    "accelerator": "CERN-LEP",
+      "collaboration": {
+      "name": "DELPHI collaboration",
+      "recid": "27050"
+    },
+    "date_created": [
+      "1993"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ],
+      "number_files": 3
```

Comment:
If we list the number of files, then it would be good to list also "size" (total in bytes for all attached files) as well, for consistency with other records.

Reply:
We can do that. Not sure what the added value is though.

Reply:
OK, I add them during merge, and fix the record links from the docs pages.

The value is basically to be consistent with data record where we display total size. This could be useful for information (how big the thing is), for size-based search facets, for enriching records with umbrella-checksum-like information...